pkg/pmem-csi-operator/utils/utils.go

```diff
@@ -47,6 +72,17 @@ func GetKubernetesVersion() (*version.Info, error) {
 	return cs.Discovery().ServerVersion()
 }
 
+// CombinedVersion returns single version number for given major and minor versions
+// The assumption is that the minor version is single dit numbers
```

Comment:
Besides the spelling ("dit"), your probably meant "at most a *double* digit number". One of your example even uses more than a single digit...

This smells like a hack. Instead of mapping a semantic version into an uint64, can you introduce a type with helper functions for version comparisons?

Reply:
> "utils" is not a good name. Ideally, this package should not exist at all.

Initially, this was started to host some utility functions like `GetKubernetesVersion()` `GetNamespace()` which are expected to be used inside the operator. I will check if we can avoid this after moving version and tls specifics to separate functions.